docs/ingress.md

```diff
@@ -33,16 +47,44 @@ Using a domain that you manage, create a DNS wildcard `A Record` pointing to the
 
 **NOTE:** you are welcome to use `*.draft.example.com` or any other wildcard domain.
 
-Remember the domain you use, it will be needed in the next step of installation as the `basedomain` passed to `draft init`.
+Remember the domain you use, it wiLl be needed in the next step of installation as the `basedomain` passed to `draft init`.
 
 | Name          | Type | Data                      |
 |---------------|------|---------------------------|
 | *.example.com | A    | `<ip address from above>` |
 
+
+### I don't manage a domain
+
+If you don't manage a domain, when you will perform the request, you can use the
+host header to use ingress the host base routing.
+
+```
+$ curl --header Host:<application domain> <ip address from above>
+```
+
+You could also edit your `/etc/hosts` file to point
+to the ingressed out application domain to your cluster.
+
+The following snippet would allow you to access an application
+
+```
+$ sudo echo <ip address from above> <application domain> >> /etc/hosts
+```
+
+The draw back is that `/etc/hosts` does not support wildcards, so you would need to
+add an entry for each. For wildcard support
+you can use `dnsmasq`. Refer to `dnsmasq` documentation for your platform.
+
+Some sources of information:
+ * [How To Setup And Configure Dnsmasq For Local Development Environment](https://www.computersnyou.com/3786/how-to-setup-dnsmasq-local-dns)
```

Comment:
Would it be okay if we link to the Arch Linux wiki here? I've always found their wiki to have a fantastic knowledge base. I'd also be more comfortable with linking to a more well-known domain.

https://wiki.archlinux.org/index.php/dnsmasq